### ISSUE TITLE
Group items in Query Log

### ISSUE THREAD

**Issue by sunindersingh** [AdguardTeam/AdGuardHome#5394]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to request a feature or enhancement and not ask a question

### Description

Hi,

Is it possible to group items in query log?

For example : There is 100 Block request for translate.googleapis.com and 17 Block request for apis.google.com. And in Query Log it shows total 117 Lines, but if there is someway to filter the log lines so it only show Group "Additional column" unique request logs. Like :-

Group Time Request Response Client
Count [100] 19:23:31 translate.googleapis.com Blocked x.x.x.x
Count [17] 19:23:31 apis.google.com Blocked x.x.x.x

So once we click on Group Column Count it expand.

Ty.

**Comment by mahmoudhamadeh** [comment]
yep, essential feature +1 for this! NexDNS can do this also can group by "Root Domains" like history.google.com + apis.google.com all grouped under *.google.com
